abis/AdminUpgradeabilityProxy.json

```diff
@@ -1,129 +0,0 @@
-[
```

Comment:
why was this file not preserved?

Reply:
is it now BridgeProxy?

Reply:
i'll add it to the abis list

Reply:
It's now ProxyAdmin and TransaprentUpgradableProxy